data_linked_in_ssr_manual/start_ssr_4_times_linux.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Does this have to be `bash`?
Would `sh` not work?

Reply:
Updated to  `sh`